content/docs/other-guides/usage-reporting.md

```diff
@@ -40,13 +49,24 @@ kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
         name: manifests
         path: common/spartakus
     name: spartakus
-```
+  ```
 
-- Delete the entry in KfDef.Spec.ComponentParams for spartakus
+- Delete the Spartakus entry in the `componentParams` section. These are the
+  lines to delete:
 
-    * initRequired: true name: usageId value: 
-    * initRequired: true name: reportUsage value:
+  ```
+  spartakus:
+  - initRequired: true
+    name: usageId
+    value: "<randomly-generated-id>"
+  - initRequired: true
+    name: reportUsage
+    value: "true"
+  ```
 
-- Delete the entry in KfDef.Spec.Components for spartakus
+- Delete the Spartakus entry in the `components` section. This is the line
```

Comment:
Same comment as above components is the old way and should largely no longer be used.

Reply:
Done.